etc/httpd/global.conf

```diff
@@ -508,6 +508,11 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/2.0/eula(?:.*) /$1legal/eula/firefox
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/3.0/eula(?:.*) /$1legal/eula/firefox-3/ [L,R=301]
 RewriteRule ^/en-US/legal/eula(.*)$ /b/en-US/legal/eula$1 [PT]
 
+# bug 1196578
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?thunderbird/about/legal/eula/$ /$1about/legal/eula/ [L,R=301]
```

Comment:
I think the trailing slash should be optional in this rule? As is, navigating to `/thunderbird/about/legal/eula` displays the existing PHP page.